src/modules/loading/utils.spec.ts

```diff
@@ -101,6 +107,11 @@ describe('modules', function() {
           const actionType = getStatus(requestAction)
           expect(actionType).toBe('REQUEST')
         })
+
+        it('should return "CLEAR" status', function() {
+          const actionType = getStatus(clearAction)
```

Comment:
Should it be actionStatus?
```suggestion
          const actionStatus = getStatus(clearAction)
```